utils/compress.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+from datetime import datetime
+import os
+import sys
+sys.path.append("..")
+import settings
+import re
+import tarfile
+import settings
+import boto3
+import logging
+
+S3_BUCKET_NAME = getattr(settings, "S3_BUCKET_NAME", None)
+S3_ACCESS_KEY_ID = getattr(settings, "S3_ACCESS_KEY_ID", None)
+S3_SECRET_ACCESS_KEY = getattr(settings, "S3_SECRET_ACCESS_KEY", None)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class Compress:
+    def __init__(self, extension):
+        self.ext = extension
+        self.secret = ""
+        self.output = None
+        self.output_fn = datetime.now().isoformat()
+        self.open()
+
+    def open(self):
+        if(re.match('^\.?tgz$', self.ext, re.IGNORECASE)):
+            try:
+                self.output = tarfile.open(self.output_fn + '.tar.gz', 'w:gz')
+            except Exception as e:
+                logger.exception('[' + self.output_fn + \
+                                 '] Unable to create output file for writing ' + str(e))
+                sys.exit(0)
```

Comment:
Let's make these `raise` so someone can catch it https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python

Pretty sure, only a `main` should exit. Not an object.

Reply:
It especially shouldn't exit with `0` since something went wrong.
I would think a better way would be `raise RuntimeError('message here')`